commands/emi.js

```diff
@@ -0,0 +1,10 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("emi")
+        .setDescription("Emi command"),
+    async execute(interation) {
+        interation.reply("This is a new EMI command.");
```

Comment:
Seems like a typo, should be interaction? Also, shouldn't you be awaiting this since it's an async function?

Reply:
Alternatively return the promise I assume is returned from .reply